dashboard/lib/metrics/events.rb

```diff
@@ -79,45 +63,38 @@ def log_event_with_session(session:, event_name:, event_value: nil, metadata: {}
       end
 
       # Builds a StatsigUser object from a user entity
-      private def build_statsig_user(user:, enabled_experiments:)
+      private def build_statsig_user(user:, enabled_experiments:, statsig_stable_id:)
+        custom_ids = {stableID: statsig_stable_id}
+
         if user.present?
-          custom_ids = {
+          custom_ids.merge!({
             user_type: user.user_type,
             enabled_experiments: enabled_experiments,
           }.compact
+)
```

Comment:
```suggestion
          custom_ids.merge!({
            user_type: user.user_type,
            enabled_experiments: enabled_experiments,
          }.compact)
```

Reply:
Interesting, Rubocop is forcing the ) to be on the next line.

What about something like this?

```
custom_ids.merge!(
    {
         user_type: user.user_type,
         enabled_experiments: enabled_experiments,
    }.compact
)
```